rcl/test/rcl/test_context.cpp

```diff
@@ -111,4 +111,8 @@ TEST_F(CLASSNAME(TestContextFixture, RMW_IMPLEMENTATION), nominal) {
   });
   EXPECT_NE(rmw_context_ptr, nullptr) << rcl_get_error_string().str;
   rcl_reset_error();
+
+  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT({
```

Comment:
Why use `OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT` here and not just call `rcl_init_options_fini` directly?

Reply:
@xabxx Friendly ping.

Reply:
@dirk-thomas you are right, no need to use the testing tool here. just revised the code. please take a look :)